From c78834f22a28c110183b885e024b40398a11a267 Mon Sep 17 00:00:00 2001
Message-Id: <c78834f22a28c110183b885e024b40398a11a267@dist-git>
From: Martin Kletzander <mkletzan@redhat.com>
Date: Wed, 31 Jan 2018 16:32:11 +0100
Subject: [PATCH] util: Rename virBitmapString to virBitmapToString

https://bugzilla.redhat.com/show_bug.cgi?id=1289368

This follows the virBitmapToData() function and, similarly to
virBitmapNewData(), we'll be able to have virBitmapNewString() later
on without name confusion.

Signed-off-by: Martin Kletzander <mkletzan@redhat.com>
Reviewed-by: John Ferlan <jferlan@redhat.com>
(cherry picked from commit 68d7cc649c652623e11e36286c9888495543dac4)
Signed-off-by: Martin Kletzander <mkletzan@redhat.com>
---
 src/libvirt_private.syms     | 2 +-
 src/qemu/qemu_capabilities.c | 4 ++--
 src/util/virbitmap.c         | 6 +++---
 src/util/virbitmap.h         | 2 +-
 4 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/libvirt_private.syms b/src/libvirt_private.syms
index 5a87276a2e..d0f2c027fc 100644
--- a/src/libvirt_private.syms
+++ b/src/libvirt_private.syms
@@ -1382,9 +1382,9 @@ virBitmapSetAll;
 virBitmapSetBit;
 virBitmapSetBitExpand;
 virBitmapSize;
-virBitmapString;
 virBitmapToData;
 virBitmapToDataBuf;
+virBitmapToString;
 
 
 # util/virbuffer.h
diff --git a/src/qemu/qemu_capabilities.c b/src/qemu/qemu_capabilities.c
index d6bd28d072..5bf188f427 100644
--- a/src/qemu/qemu_capabilities.c
+++ b/src/qemu/qemu_capabilities.c
@@ -1525,7 +1525,7 @@ int virQEMUCapsParseHelpStr(const char *qemu,
                                    qemuCaps, check_yajl) < 0)
         goto cleanup;
 
-    strflags = virBitmapString(qemuCaps->flags, true);
+    strflags = virBitmapToString(qemuCaps->flags, true);
     VIR_DEBUG("Version %u.%u.%u, cooked version %u, flags %s",
               major, minor, micro, *version, NULLSTR(strflags));
     VIR_FREE(strflags);
@@ -2407,7 +2407,7 @@ virQEMUCapsClear(virQEMUCapsPtr qemuCaps,
 
 char *virQEMUCapsFlagsString(virQEMUCapsPtr qemuCaps)
 {
-    return virBitmapString(qemuCaps->flags, true);
+    return virBitmapToString(qemuCaps->flags, true);
 }
 
 
diff --git a/src/util/virbitmap.c b/src/util/virbitmap.c
index f08384ff1c..e7d04e8c8b 100644
--- a/src/util/virbitmap.c
+++ b/src/util/virbitmap.c
@@ -310,7 +310,7 @@ int virBitmapGetBit(virBitmapPtr bitmap, size_t b, bool *result)
 }
 
 /**
- * virBitmapString:
+ * virBitmapToString:
  * @bitmap: Pointer to bitmap
  * @prefix: Whether to prepend "0x"
  *
@@ -319,8 +319,8 @@ int virBitmapGetBit(virBitmapPtr bitmap, size_t b, bool *result)
  * Returns pointer to the string or NULL on error.
  */
 char *
-virBitmapString(virBitmapPtr bitmap,
-                bool prefix)
+virBitmapToString(virBitmapPtr bitmap,
+                  bool prefix)
 {
     virBuffer buf = VIR_BUFFER_INITIALIZER;
     size_t sz;
diff --git a/src/util/virbitmap.h b/src/util/virbitmap.h
index dc8fb71a07..99eb779d71 100644
--- a/src/util/virbitmap.h
+++ b/src/util/virbitmap.h
@@ -80,7 +80,7 @@ bool virBitmapIsBitSet(virBitmapPtr bitmap, size_t b)
 int virBitmapGetBit(virBitmapPtr bitmap, size_t b, bool *result)
     ATTRIBUTE_NONNULL(1) ATTRIBUTE_NONNULL(3) ATTRIBUTE_RETURN_CHECK;
 
-char *virBitmapString(virBitmapPtr bitmap, bool prefix)
+char *virBitmapToString(virBitmapPtr bitmap, bool prefix)
     ATTRIBUTE_NONNULL(1) ATTRIBUTE_RETURN_CHECK;
 
 char *virBitmapFormat(virBitmapPtr bitmap);
-- 
2.16.1

